src/libraries/System.Collections/tests/Generic/CollectionExtensionsTests.cs

```diff
@@ -134,6 +143,13 @@ public void AsReadOnly_NullIList_ThrowsArgumentNullException()
             Assert.Throws<ArgumentNullException>("list", () => list.AsReadOnly());
         }
 
+        [Fact]
+        public void AsReadOnly_NullISet_ThrowsArgumentNullException()
+        {
+            ISet<string> set = null;
+            Assert.Throws<ArgumentNullException>("set", () => set.AsReadOnly());
```

Comment:
Nit:
```suggestion
            AssertExtensions.Throws<ArgumentNullException>("set", () => set.AsReadOnly());
```
Some size-focused builds may strip out the parameter name.